packages/rest/src/validation/ajv-factory.provider.ts

```diff
@@ -55,34 +53,29 @@ export class AjvFactoryProvider implements Provider<AjvFactory> {
       // See https://github.com/epoberezkin/ajv#options
       validationOptions = {
         allErrors: true,
-        jsonPointers: true,
-        // nullable: support keyword "nullable" from Open API 3 specification.
-        nullable: true,
+        strictTypes: false,
```

Comment:
how this will affect the OpenApi 3 Spec?

Reply:
`nullable` is supported by default now for AJV 7.x & 8.x.